### PR TITLE
pattern for CDDL+GPL License

### PR DESCRIPTION
**Pull request by lumber1000**

### CODE REVIEW

File: license-compliance/gradle-license-report/allowed-licenses.json [hunk 1]
```diff
@@ -549,6 +549,9 @@
     },
     {
       "moduleLicense": "^MIT OR MPL-2.0$"
+    },
+    {
+      "moduleLicense": "^CDDL\\+GPL License$"
```
Comment (OptimumCode) [inline license-compliance/gradle-license-report/allowed-licenses.json:554]: I think we should make this change a bit differently. We should add a normalization that transforms this license into "CDDL OR GPL" (please find corresponding version for those licenses) and add this normalized combination to allowed list

**Review by OptimumCode — CHANGES_REQUESTED** [review]

**Comment by OptimumCode** [comment]
@lumber1000 Do you have any updates on this?

File: license-compliance/gradle-license-report/license-normalizer-bundle.json [hunk 2]
```diff
@@ -582,6 +567,10 @@
     {
       "bundleName": "mit-or-mpl-2.0",
       "licenseNamePattern": "^MIT License; Mozilla Public License 2\\.0 \\(MPL 2\\.0\\)$"
+    },
+    {
+      "bundleName": "cddl-1.1-or-gpl-2",
```
Comment (OptimumCode) [inline license-compliance/gradle-license-report/license-normalizer-bundle.json:572]: question: Have you checked this normalization locally? In the report you shared there was no license URL. I am not sure if this pattern will work in this case

Comment (lumber1000) [reply license-compliance/gradle-license-report/license-normalizer-bundle.json:572]: Yes, I've checked: plugin recognizes this license. This URL is located in the MANIFEST.MF file.

**Review by OptimumCode — APPROVED** [review]

**Pull request merged**